quiche/src/lib.rs

```diff
@@ -4805,6 +4804,37 @@ impl Connection {
         MIN_CLIENT_INITIAL_LEN
     }
 
+    /// Queue a PING frame to be sent for the active path if no other
+    /// ACK-eliciting frame would otherwise be sent. See
+    /// [`send_ack_eliciting_on_path()`] for more detail.
+    ///
+    /// [`send_ack_eliciting_on_path()`]:
+    /// struct.Connection.html#method.send_ack_eliciting_on_path
```

Comment:
Our usual pattern for public docs is a simple one line description on first line, then more detail on subsequent lines. So maybe something like 

```
/// Schedule an ack-eliciting packet on the active path.
///
/// QUIC packets might not contain ack-eliciting frames during normal operating conditions.
/// If the packet would already contain ack-eliciting frames, this method does not change any behavior.
/// However, if the packet would not ordinarily contain ack-eliciting frames, this method ensures that a PING frame
/// sent.
///
/// Calling this method multiple times before [`send()`] has no effect.
```